### PR TITLE
YutakaSawada.MultiPar version 1.3.2.6

### PR DESCRIPTION
**Pull request by FinnKrestel**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94128)

### CODE REVIEW

File: manifests/y/YutakaSawada/MultiPar/1.3.2.6/YutakaSawada.MultiPar.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: YutakaSawada.MultiPar
```
Comment (mdanish-kh) [inline manifests/y/YutakaSawada/MultiPar/1.3.2.6/YutakaSawada.MultiPar.installer.yaml:4]:
```suggestion
PackageIdentifier: YutakaSawada.MultiPar.Beta
```
This release is a beta version.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by mdanish-kh** [comment]
After committing the above changes, you would also have to modify the directory structure to match the new PackageIdentifier. `manifests/y/YutakaSawada/Multipar/1.3.2.6` should be changed to `manifests/y/YutakaSawada/Multipar/Beta/1.3.2.6`. Additionally, the yaml file names would also require renaming to match the new PackageId. `YutakaSawada.MultiPar.installer.yaml` -> `YutakaSawada.MultiPar.Beta.installer.yaml` and so on.

A quick way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key that'll open this PR in VSCode Online. Navigate to explorer tab on the left, find the required files and perform the suggested operations. Afterwards you can stage and commit your changes from the Source Control tab on the left (the changes are automatically pushed).

File: manifests/y/YutakaSawada/MultiPar/1.3.2.6/YutakaSawada.MultiPar.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: YutakaSawada.MultiPar
+PackageVersion: 1.3.2.6
+PackageLocale: en-US
+Publisher: Yutaka Sawada
+PackageName: MultiPar
+License: GPL
```
[the remark this answers is not in the document]
Comment (FinnKrestel) [reply manifests/y/YutakaSawada/MultiPar/1.3.2.6/YutakaSawada.MultiPar.locale.en-US.yaml:9]: Looks like the license has been changed from GPL to an undisclosed one. I too can't find any evidence for the project still using GPL as the main licensing model, so I'll change it. Thanks for mentioning! ;)

File: manifests/y/YutakaSawada/MultiPar/Beta/1.3.2.6/YutakaSawada.MultiPar.Beta.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: YutakaSawada.MultiPar.Beta
+PackageVersion: 1.3.2.6
+PackageLocale: en-US
+Publisher: Yutaka Sawada
+PackageName: MultiPar
+License: Proprietary
```
Comment (mdanish-kh) [inline manifests/y/YutakaSawada/MultiPar/Beta/1.3.2.6/YutakaSawada.MultiPar.Beta.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
LicenseUrl: https://hp.vector.co.jp/authors/VA021385/archive.htm
```
Found this on the author's page, says the GUI app is closed source. We can link to the license here.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/y/YutakaSawada/MultiPar/Beta/1.3.2.6/YutakaSawada.MultiPar.Beta.locale.en-US.yaml (hunk 3 above)
Comment (FinnKrestel) [reply manifests/y/YutakaSawada/MultiPar/Beta/1.3.2.6/YutakaSawada.MultiPar.Beta.locale.en-US.yaml:9]: Alright.

**Review by mdanish-kh — APPROVED** [review]

**Comment by FinnKrestel** [comment]
Maybe the path for v1.3.2.4 should be changed as well since it's also a beta version. The path also looked a little bit weird to me, but I wanted to keep the seemingly correct (or what I thought it was) naming convention of the other version. Maybe the stable version should also be made available in the package definitions?

**Comment by mdanish-kh** [comment]
> Maybe the path for v1.3.2.4 should be changed as well since it's also a beta version. The path also looked a little bit weird to me, but I wanted to keep the seemingly correct (or what I thought it was) naming convention of the other version. Maybe the stable version should also be made available in the package definitions?

Yep I noticed that as well. I'll update the YutakaSawada.MultiPar manifest with the stable release of `1.3.1.9`

**Review by ghost — APPROVED** [review]

**Pull request merged**